Mathlib/CategoryTheory/Adjunction/Basic.lean

```diff
@@ -419,6 +420,22 @@ def ofNatIsoRight {F : C ⥤ D} {G H : D ⥤ C} (adj : F ⊣ G) (iso : G ≅ H)
   Adjunction.mkOfHomEquiv
     { homEquiv := fun X Y => (adj.homEquiv X Y).trans (equivHomsetRightOfNatIso iso) }
 
+/-- The isomorpism which an adjunction `F ⊣ G` induces on `G ⋙ yoneda`. This states that
+`Adjunction.homEquiv` is natural in both arguments. -/
+@[simps!]
+def yonedaIso {C : Type u₁} [Category.{v₁} C] {D : Type u₂} [Category.{v₁} D]
```

Comment:
```suggestion
def compYonedaIso {C : Type u₁} [Category.{v₁} C] {D : Type u₂} [Category.{v₁} D]
```
seems a bit more helpful imo, since comp appears on the left. Or, we could swap the iso around and talk about `yonedaCompIso`?

Reply:
I like the current direction more because it has the non-whiskered functor on the left-hand side, I'll just rename it